engine/src/org/pentaho/di/trans/steps/csvinput/CsvInputMeta.java

```diff
@@ -125,7 +125,7 @@ public void setDefault() {
     delimiter = ",";
     enclosure = "\"";
     headerPresent = true;
-    lazyConversionActive = true;
+    lazyConversionActive = false;
```

Comment:
While I'd like to see the above become the default, I don't think we can make that change within the scope of this bug.